examples/gep2257.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python3
+# -*- coding:utf-8 -*-
+
+"""
+This example uses kubernetes.utils.duration to parse and display
+a GEP-2257 duration string (you can find the full specification at
+https://gateway-api.sigs.k8s.io/geps/gep-2257/).
+
+Good things to try:
+>>> python examples/gep2257.py 1h
```

Comment:
nit: could you change the file name to something like duration-gep2257, to make it more clear what it is about ?

Reply:
Done! 🙂